src/cli-commands/tools.ts

```diff
@@ -4,6 +4,13 @@ import { getToolsRoot } from "../util";
 import { Config } from "../persistent-types";
 
 export default async (config: Config) => {
+  if (!config["config-file"]) {
+    console.warn(
+      `This command works best when with a configuration file (config.json).
+See docs/configuration.md for details.`
```

Comment:
```suggestion
See docs/configuration.md and docs/analysis-tools.md for details.`
```

In the error message in `run.ts` you refer to both `configuration.md` and `analysis-tools.md`, I think you should do the same here.